helm-framework/helm/provider.go

```diff
@@ -579,7 +583,7 @@ func (p *HelmProvider) Configure(ctx context.Context, req provider.ConfigureRequ
 
 	meta := &Meta{
 		Data: &HelmProviderModel{
-			Debug:                types.BoolValue(debug == "true"),
```

Comment:
I believe there was one other place that had it set as `debug == "true"`. Could you double check just in case? I may just be misremembering also.

Reply:
Just did a GREP and didn't see it used anywhere else in the codebase. Also I believe it would throw an error if it was somewhere else I believe!